src/main/resources/static/api.yml

```diff
@@ -1823,3 +2006,93 @@ components:
         - type
         - subType
         - note
+    Assessment:
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+        application:
+            $ref: '#/components/schemas/Application'
+        assignedToProbationOfficer:
+          type: string
+          format: uuid
+        schemaVersion:
+          type: string
+          format: uuid
+        outdatedSchema:
+          type: boolean
+        createdAt:
+          type: string
+          format: date-time
+        assignedAt:
+          type: string
+          format: date-time
+        submittedAt:
+          type: string
+          format: date-time
+        data:
+          $ref: '#/components/schemas/AnyValue'
+        clarificationNotes:
+          type: array
+          items:
+            $ref: '#/components/schemas/ClarificationNote'
+      required:
+        - id
+        - application
+        - assignedToProbationOfficer
+        - schemaVersion
+        - outdatedSchema
+        - createdAt
+        - assignedAt
+        - data
+        - clarificationNotes
+    UpdateAssessment:
+      type: object
+      properties:
+        data:
+          type: object
+          additionalProperties:
+            $ref: '#/components/schemas/UpdateAssessment'
```

Comment:
Is this going to cause an infinite loop?

Reply:
Ah right you are, was meant to be AnyValue